contracts/svm/cli/src/main.rs

```diff
@@ -0,0 +1,341 @@
+    pub fee_receiver_relayer: Option<Pubkey>,
+
+    #[arg(long, default_value = "4000")]
+    /// The portion of the bid that goes to the router, in bps
+    pub split_router_default: u64,
+
+    #[arg(long, default_value = "2000")]
+    /// The portion of the bid (after router fees) that goes to relayer, in bps
+    pub split_relayer: u64,
+}
+
+#[derive(Debug, Subcommand)]
+enum Commands {
+    Initialize(Initialize),
+    SetAdmin {
+        #[arg(long)]
+        /// Path to the private key json file for the current admin
+        /// This account will be used as the transaction payer as well
+        admin: String,
+
+        #[arg(long)]
+        /// New admin pubkey
+        admin_new: Pubkey,
+    },
+    SetRelayer {
+        #[arg(long)]
+        /// Path to the private key json file for the admin
+        /// This account will be used as the transaction payer as well
+        admin: String,
+
+        #[arg(long)]
+        /// Signer used for relaying the bids
+        relayer_signer: Pubkey,
+
+        #[arg(long)]
+        /// Fee receiver for the relayer
+        fee_receiver_relayer: Pubkey,
+    },
+    SetSplits {
+        #[arg(long)]
+        /// Path to the private key json file for the admin
+        /// This account will be used as the transaction payer as well
+        admin: String,
+
+        #[arg(long)]
+        /// The portion of the bid that goes to the router, in bps
+        split_router_default: u64,
+
+        #[arg(long)]
+        /// The portion of the bid (after router fees) that goes to relayer, in bps
+        split_relayer: u64,
+    },
+    SetRouterSplit {
+        #[arg(long)]
+        /// Path to the private key json file for the admin
+        /// This account will be used as the transaction payer as well
+        admin: String,
+
+        #[arg(long)]
+        /// The pubkey of the router to set the split for
+        router: Pubkey,
+
+        #[arg(long)]
+        /// The split to use for this specific router, in bps
+        split_router: u64,
+    },
+    WithdrawFees {
+        #[arg(long)]
+        /// Path to the private key json file for the admin
+        /// This account will be used as the transaction payer as well
+        admin: String,
+
+        #[arg(long)]
+        /// The pubkey of the account that receives the fees
+        fee_receiver: Pubkey,
+    },
+    GetConfig,
+    GetRouterSplit {
+        #[arg(long)]
+        /// The pubkey of the router to get the split for
+        router: Pubkey,
+    },
+}
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let args = Args::parse();
+    let program_id = args.program_id.unwrap_or(express_relay::ID);
+    let express_relay_metadata = Pubkey::find_program_address(&[SEED_METADATA], &program_id).0;
+    let cluster = Cluster::from_str(args.rpc_url.as_str())?;
+
+    match args.command {
+        Commands::Initialize(init_args) => {
+            let payer = read_keypair_file(init_args.payer)?;
```

Comment:
Not sure about this, but you can implement a function like:
```
fn send_transaction(cluster: Cluster, payer_file: String, program_id: Pubkey, accounts: impl ToAccountMetas, args: impl InstructionData) -> Result<Signature, Box<dyn std::error::Error>> {
    let payer = read_keypair_file(payer_file)?;
    let client = Client::new_with_options(cluster, &payer, CommitmentConfig::confirmed());
    let program = client.program(program_id)?;
    let result = program
        .request()
        .accounts(accounts)
        .args(args)
        .signer(&payer)
        .send()?;
    Ok(result)
}
```

and then extract the result with:

```
let result = send_transaction(
                cluster,
                init_args.payer,
                program_id,
                accounts::Initialize {
                    payer: Pubkey::default(),
                    express_relay_metadata,
                    admin: Pubkey::default(),
                    relayer_signer: Pubkey::default(),
                    fee_receiver_relayer: Pubkey::default(),
                    system_program: system_program::ID,
                },
                instruction::Initialize {
                    data: InitializeArgs {
                        split_router_default: init_args.split_router_default,
                        split_relayer:        init_args.split_relayer,
                    },
                },
            )?;
```

I think this way we can get rid of bunch of duplicate codes :-?

Reply:
the issue is we still need the payer pubkey when creating the `accounts::Initialize` object.
I tried to make it a bit simpler